tests/gamma/csm_observability_test.py

```diff
@@ -446,6 +343,109 @@ def test_csm_observability(self):
                     all_results[metric].points, RESPONSE_PAYLOAD_SIZE
                 )
 
+    # These parameters are more pertaining to the test itself, not to
+    # each run().
+    def initKubernetesClientRunner(self, **kwargs) -> KubernetesClientRunner:
+        return super().initKubernetesClientRunner(
+            csm_workload_name=CSM_WORKLOAD_NAME_CLIENT,
+            csm_canonical_service_name=CSM_CANONICAL_SERVICE_NAME_CLIENT,
+        )
+
+    # These parameters are more pertaining to the test itself, not to
+    # each run().
+    def initKubernetesServerRunner(self, **kwargs) -> GammaServerRunner:
+        return super().initKubernetesServerRunner(
+            csm_workload_name=CSM_WORKLOAD_NAME_SERVER,
+            csm_canonical_service_name=CSM_CANONICAL_SERVICE_NAME_SERVER,
+        )
```

Comment:
We should keep these two at the top, after `setUpClass`, but before the test. These are method overrides, and are not explicitly called from the test. That's why I want to keep them visible.

Reply:
Fixed.